docs/src/misc.rst

```diff
@@ -522,6 +522,16 @@ API
     Gets the executable path. You *must* call `uv_setup_args` before calling
     this function.
 
+.. c:function:: int uv_exepath2(char* buffer, size_t* size)
+
+  Like :c:func:`uv_exepath`, but returns :c:macro:`UV_ENOBUFS` if ``buffer``
+  is too small and sets ``*size`` to the required length (including the null
+  terminator). Returns :c:macro:`UV_ENOTSUP` on unsupported platforms.
```

Comment:
Can you clarify here that the return value of a successful call does  **not** include the nul byte?

Minor aside: it's nul or NUL, not null, but I've made peace with the fact that everyone and his dog uses "null byte" nowadays.